test/testMethods.ts

```diff
@@ -217,6 +217,13 @@ module TestMethods {
     target.node().dispatchEvent( e );
   }
 
+  export function triggerFakeKeyboardEvent(type: string, target: d3.Selection<void>, keyCode: number) {
+    var event = <KeyboardEvent> document.createEvent("Events");
+    event.initEvent(type, true, true);
+    event.keyCode = keyCode;
+    target.node().dispatchEvent( event );
```

Comment:
It's weird we have no tslint rule around this. Can you remove the whitespace around `(` and `)`